src/combine.cpp

```diff
@@ -5940,22 +5906,27 @@ static void handle_oc_property_decl(chunk_t *os)
                   }
                   setter_chunks.push_back(chunkGroup);
                }
-               else if (chunk_is_str(next, "nullable", 8))
+               else if (chunk_is_str(next, "nullable", 8) ||
+                        chunk_is_str(next, "nonnull", 7) ||
+                        chunk_is_str(next, "null_resettable", 15) ||
+                        chunk_is_str(next, "null_unspecified", 16)
+                        )
```

Comment:
@gmaurel do we need a new rule in forUncrustifySources.cfg? I think that this paren should not be on a new line.